### ISSUE TITLE
Tabs component's tablist does not need to be in a nav

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon#2272]
Please see https://github.com/IBM/carbon-components-react/issues/2171

**Comment by alisonjoseph** [comment]
not stale

**Comment by carmacleod** [comment]
@joshblack Is this really fixed? I didn't see a commit go by that fixed it. The above commit just changes a comment somewhere - it may be referencing the wrong issue number?

The [Vanilla Tabs](https://www.carbondesignsystem.com/components/tabs/code) and [React Tabs](http://react.carbondesignsystem.com/?selectedKind=Tabs&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel) example pages still have a nav as a parent for the tab list, so I think (?) that this issue was closed accidentally? Unless those example pages still need to be updated?

**Edit**: I think what happened is that the comment fix in the above commit somehow referenced this issue (Vanilla 2272) but the fix was actually for [React 2272](https://github.com/carbon-design-system/carbon-components-react/issues/2272). Then, some kind of tooling thought it could close this issue, but it needed to only close the react issue of the same number.
